client/elife-theme/src/elements/index.js

```diff
@@ -1,3 +1,11 @@
 export { default as Button } from './Button'
 export { default as Heading } from './Heading'
 export * from './Heading'
+
+export {
+  default as StepsRoot,
```

Comment:
Why not name the export `StepsRoot` then you can just `export *`?

Reply:
Isn't `export *` the spawn of Satan?

Reply:
This is a re-export. It's only exporting the things which were imported from `Steps`. JavaScript doesn't have a way to export all variables (for the very good reason you mention).